app/src/main/java/d/d/meshenger/AddressActivity.kt

```diff
@@ -7,16 +7,13 @@ import android.content.ServiceConnection
 import android.graphics.Color
 import android.os.Bundle
 import android.os.IBinder
-import android.text.Editable
-import android.text.TextWatcher
```

Comment:
Let's only do one thing in this committ. Please remove these import cleanups. Thank you.